packages/mdc-textfield/foundation.js

```diff
@@ -186,6 +192,25 @@ class MDCTextFieldFoundation extends MDCFoundation {
     }
   }
 
+  /**
+   * Changes the floating label's state
+   * @private
+   */
+  updateLabelFloat_() {
+    const input = this.getNativeInput_();
+    const {LABEL_FLOAT_ABOVE} = MDCTextFieldFoundation.cssClasses;
+
```

Comment:
add a

```
const labelFloatIsActive = this.adapter_.labelHasClass(LABEL_FLOAT_ABOVE);
```

Then reference `labelFloatIsActive` instead of `this.labelFloatIsActive_` for the rest of this method